code/__HELPERS/level_traits.dm

```diff
@@ -10,13 +10,13 @@ GLOBAL_VAR(station_level_z_scratch)
 
 // Called a lot, somewhat slow, so has its own cache
 #define is_station_level(z) \
-	( \
+	( ##z && \
```

Comment:
Why are you using ##z instead now? ##var is for using the argument as a variable name, it doesn't need ## if its just being used as a variable.
![image](https://github.com/BeeStation/BeeStation-Hornet/assets/26465327/fe418c6e-e0c4-49c4-b6fb-de52bf4fd1ee)

Reply:
oh, you're right. I was confused with something for a while.